services/MerchStoreService.ts

```diff
@@ -214,6 +221,15 @@ export default class MerchStoreService {
         throw new NotFoundError(`Missing: ${missingItems}`);
       }
 
+      // Checks that hidden items were not ordered
+      const hiddenItem = Array.from(itemOptions.values())
+        .filter((o) => o.item.hidden)
+        .map((o) => o.uuid);
+
+      if (hiddenItem.length !== 0) {
+        throw new ForbiddenError(`Not allowed to order: ${hiddenItem}`);
+      }
+
```

Comment:
Let's give a bit more detail to why the user is not allowed to order the items - something like:
```ts
throw new ForbiddenError(`The following items are no longer available: ${hiddenItems}`)
```

Reply:
`ForbiddenError` is usually reserved for permissions the user doesn't have, like accessing an admin-only route. This seems more like a `UserError` since the user placed an invalid order.